modules/deploy/partials/kubernetes/guides/uninstall.adoc

```diff
@@ -0,0 +1,32 @@
+= Uninstall Redpanda
+
+When you've finished testing Redpanda, you can uninstall it from your cluster and delete any Kubernetes resources that the Helm chart created.
+
+[tabs]
+======
+Helm + Operator::
++
+--
+[,bash]
+----
+kubectl delete -f redpanda-cluster.yaml --namespace <namespace>
+helm uninstall redpanda-controller --namespace <namespace>
+kubectl delete pod --all --namespace <namespace>
```

Comment:
I'm not sure why we need to delete Pods. Did you saw Pods left behind?

Reply:
The configurator Pods get left behind